src/docs/truffle/testing/testing-your-contracts.md

```diff
@@ -8,14 +8,14 @@ layout: docs.hbs
 
 Truffle comes standard with an automated testing framework to make testing your contracts a breeze. This framework lets you write simple and manageable tests in two different ways:
 
-* In [Javascript](/docs/getting_started/javascript-tests), for exercising your contracts from the outside world, just like your application.
+* In [Javascript](/docs/getting_started/javascript-tests) and [Typescript](/docs/getting_started/typescript-tests), for exercising your contracts from the outside world, just like your application.
```

Comment:
Huh did these paths get changed somewhere? It doesn't seem to correspond to the local path. Maybe I'm missing something

Reply:
looks like the paths are there for web navigation